app/templates/components/settings-modal.hbs

```diff
@@ -0,0 +1,24 @@
+<div class="modal-background" onclick={{action "hideModal"}}></div>
+<div class="modal-content">
+  <div class="modal-card">
+    <header class="modal-card-head">
+      <p class="modal-card-title">User Settings</p>
+      <button class="delete" aria-label="close" onclick={{action "hideModal"}}></button>
+    </header>
+    <section class="modal-card-body">
+      <div class="field">
+        <label class="label">GitHub Token</label>
+        <div class="control">
+          {{input class="input" value=token_github_com}}
+        </div>
+        <p class="help is-danger">In order to fetch results from GitHub, you must specify GitHub token.</p>
```

Comment:
What scopes are in need? Some people are reluctant to authorize access to all scopes.
ref: https://developer.github.com/apps/building-oauth-apps/understanding-scopes-for-oauth-apps/

Reply:
Any of it. Except GitHub made some updates.

Reply:
The exact scopes that are needed should be listed here.